package/Dockerfile

```diff
@@ -1,13 +1,22 @@
-FROM registry.suse.com/bci/bci-base:15.3
+FROM ghcr.io/oracle/oraclelinux:7-slim
 
 ARG KUBERNETES_RELEASE=v1.21.12
 WORKDIR /usr/local/bin
-RUN set -x \
- && zypper -n install curl \
- && curl -fsSLO https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_RELEASE}/bin/linux/amd64/kubectl \
- && chmod +x kubectl
+
+RUN yum-config-manager --enable ol7_optional_latest && \
```

Comment:
Use a single RUN command to reduce number of layers

Reply:
fixed